etl/tests/common/pipeline_v2.rs

```diff
@@ -1,12 +1,20 @@
 use etl::v2::config::batch::BatchConfig;
 use etl::v2::config::pipeline::PipelineConfig;
+use etl::v2::config::retry::RetryConfig;
 use etl::v2::destination::base::Destination;
 use etl::v2::pipeline::{Pipeline, PipelineIdentity};
 use etl::v2::state::store::base::StateStore;
 use postgres::tokio::options::PgDatabaseConfig;
+use rand::random;
+use std::time::Duration;
+
+pub fn create_pipeline_identity(publication_name: &str) -> PipelineIdentity {
```

Comment:
We need a random pipeline id since slot names are generated based on the pipeline id and we want to avoid clashes.